core/src/main/resources/org/apache/spark/ui/static/structured-streaming-page.js

```diff
@@ -106,12 +106,12 @@ function drawAreaStack(id, labels, values, minX, maxX, minY, maxY) {
         .on('mouseover', function(d) {
             var tip = '';
             var idx = 0;
-            var _values = timeToValues[d._x]
+            var _values = formattedTimeToValues[d._x]
```

Comment:
I've also renamed to add `formatted`, as it requires formatted time string for x axis of input.